test/test_torch.py

```diff
@@ -2581,13 +2581,15 @@ def test_batch_norm_cpu_inference(self):
                     self.assertEqual(output3, output1)
                     self.assertEqual(output3, output2)
 
+        @noarchTest
```

Comment:
Will there also be a PR going through every test in the codebase so far to mark some as noarch or will it be more of a gradual "from now on" thing?

Reply:
It'll be as needed. The main motivating use case for this is I'm adding 17k new tests in #53682 and I don't want them to run on all CI runs.